setup.sh

```diff
@@ -2,7 +2,12 @@
 
 set -e
 sudo apt update
-sudo apt install -y python3-pip python3-pexpect unzip busybox-static fakeroot kpartx snmp uml-utilities util-linux vlan qemu-system-arm qemu-system-mips qemu-system-x86 qemu-utils lsb-core wget tar
+sudo apt install -y python3-pip python3-pexpect unzip busybox-static fakeroot kpartx snmp uml-utilities util-linux vlan qemu-system-arm qemu-system-mips qemu-system-x86 qemu-utils wget tar
+
+if [ ! -x $(which lsb_release) ]
```

Comment:
Can you put quotes around? Like `if [ ! -x "$(which lsb_release)" ]`

Else it doesn't work while testing on a docker ubuntu:18.04 container which doesn't come with lsb_release out of the box.

Reply:
Updated